frontend/packages/knative-plugin/src/components/add/import-types.ts

```diff
@@ -16,6 +17,7 @@ export const EventSources = {
   KafkaSource: EventSourceKafkaModel.kind,
   PingSource: EventSourcePingModel.kind,
   SinkBinding: EventSourceSinkBindingModel.kind,
+  KameletBinding: CamelKameletBindingModel.kind,
```

Comment:
Do we need this?

Reply:
yes, I am using this here https://github.com/openshift/console/pull/7344/files#diff-eb8c1eb7c82877b06a83a575484c7ba373df71b7ce39c7d1fa2e776e11f9edecR101 in switch case.